### PR TITLE
Refactorize visualization Capacity and Infrastructure

### PR DESCRIPTION
**Pull request by nicolas-brousse**
- [x] Implement form
- [x] Implement turbo frame or stream
- [x] Re-Implement LeaderLine
- [ ] Cleanup code view
- [ ] Try to optimize SQL queries / add cache
- [x] Remove old code
- [ ] Move some element in components?
- [x] Tests

### CODE REVIEW

File: app/controllers/visualization/infrastructures_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    def load_data!
+      @islet = Islet.find(@filter.islet_id)
+      @room = @islet.room
+
+      @servers = Server.includes(:frame, :stack, :ports, :cards => [:ports])
+        .where.not(network_types: [])
+      # .includes(:cards, :ports => [:connection => [:port, :cable =>[:connections => [:port => :card]]]]).
+      @concentrateurs_ids = [383, 384, 1043, 1044]
+      @concentrateurs = Server.where(id: @concentrateurs_ids).includes(:ports => :connection, :cards => [:ports => :connection])
```
Comment (B-Rass) [inline app/controllers/visualization/infrastructures_controller.rb]: Don't we want to remove those `=>`

File: app/helpers/servers_helper.rb [hunk 2]
```diff
@@ -146,7 +146,7 @@ def link_to_port_by_type(label, type, port_data, position, card_id, port_id)
 
     link_to label.to_s, edit_port_url, id: port_id, title: (port_data.present? ? "#{port_data.vlans}" : ""),
                                        class: "border border-secondary port port#{port_class} #{port_data.try(:cable_color) ? port_data.try(:cable_color) : "empty"}",
-                                       data: { url: edit_port_url, position: position, type: type, controller: 'tooltip', bs_placement: 'top' }
+                                       data: { url: edit_port_url, position: position, type: type, controller: 'tooltip', bs_placement: 'top' }, target: :_top
```
Comment (B-Rass) [inline app/helpers/servers_helper.rb]:
```suggestion
                                       data: { url: edit_port_url, position: , type: , controller: 'tooltip', bs_placement: 'top' }, target: :_top
```

**Review by B-Rass — CHANGES_REQUESTED** [review]

File: app/controllers/visualization/infrastructures_controller.rb (hunk 1 above)
Comment (nicolas-brousse) [reply app/controllers/visualization/infrastructures_controller.rb]: We could

File: app/views/visualization/infrastructures/show.html.erb
Comment (B-Rass) [inline app/views/visualization/infrastructures/show.html.erb:1]: Remove the not translated string here ("Appliquer", "Loading..")

File: app/views/visualization/network_capacities/show.html.erb
Comment (B-Rass) [inline app/views/visualization/network_capacities/show.html.erb:1]: same here

**Review by B-Rass — CHANGES_REQUESTED** [review]

**Review by B-Rass — APPROVED** [review]

**Pull request merged**